beets/autotag/mb.py

```diff
@@ -223,6 +223,8 @@ def track_info(recording, index=None, medium=None, medium_index=None,
     if recording.get('length'):
         info.length = int(recording['length']) / (1000.0)
 
+    info.trackdisambig = recording.get('disambiguation', None)
```

Comment:
This seems like a perfectly fine way to do it, but it's exactly equivalent to `recording.get('disambiguation')` (the default default value is `None`!).